ADAL/src/validation/ADAuthorityValidation.m

```diff
@@ -326,6 +326,25 @@ - (NSURL *)networkUrlForAuthority:(NSURL *)authority
     return url;
 }
 
+- (NSURL *)cacheUrlForAuthority:(NSURL *)authority
+                        context:(id<ADRequestContext>)context
+{
+    if ([ADHelpers isADFSInstanceURL:authority])
+    {
+        return authority;
+    }
+    
+    NSURL *url = [_aadCache cacheUrlForAuthority:authority];
+    if (!url)
+    {
+        AD_LOG_WARN(@"No cached preferred_cache for authority", context.correlationId, nil);
+        return authority;
+    }
+    
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS1131)